api/proto/info/info.proto

```diff
@@ -278,6 +278,20 @@ message ZCellularModuleInfo {
     string firmware_version = 3;
 }
 
+// Device Run State
```

Comment:
Why do we eve need to have "Run" be part of the name? It is the DeviceState as far as I'm concerned.

Reply:
was thinking I could keep the name of the attribute run_state for all objects. Will just stick with "state" then:

ZDeviceState state = 44;